src/ripple/server/impl/BaseWSPeer.h

```diff
@@ -43,6 +43,12 @@ class BaseWSPeer
     using BasePeer<Impl>::fail;
     using BasePeer<Impl>::strand_;
 
+    enum
+    {
+        // Max seconds without completing a message
+        timeoutSeconds = 30
```

Comment:
Personally, I wouldn't put this in the `enum`. It belongs in the function that sets the timeout, i.e. inside `start_timer`.